README.md

```diff
@@ -8,8 +8,34 @@ Level Up is a Steam game recommendations site where users can submit their own a
 
 ## Run it yourself
 
-Level Up's server components are written in Go and operate on top of an Amazon DynamoDB database (DynamoDB's free tier is more than enough to host).
+### 1. using Docker
+
+Make sure you have the .env file if not run: `cp .env.example .env && $EDITOR .env`
+```
+`docker build -t levelup .` 
+
+`docker run --env-file .env -p 4000:4000 levelup` 
+```
+
+or
+
+```
+`docker run --env-file .env -p 4000:4000 bikerwhocodes/levelup` 
```

Comment:
I do plan on publishing to Docker Hub immediately after I merge this using GitHub Actions, so this can probably be safely changed to `antigravities/levelup`.